extensions/kubernetes-client/deployment-internal/pom.xml

```diff
@@ -34,6 +34,21 @@
             <groupId>io.fabric8</groupId>
             <artifactId>kubernetes-client</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-devservices-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.dajudge.kindcontainer</groupId>
+            <artifactId>kindcontainer</artifactId>
+            <version>1.3.0</version>
```

Comment:
I think all the versions we provide should be declared in `bom/application/pom.xml`

Reply:
Does this work on all platforms?

Reply:
Yes, but AFAIK it can't be tested with windows container, so test should be disable in Windows test profile.

Reply:
> Apart from my comments, I think we should cover the new dev services as part of the existing `integration-tests/kubernetes-client` module. There is an internal testing framework we can use for this (see `@QuarkusDevModeTest`).

@Sgitario `integration-tests/kubernetes-client` use `quarkus-test-kubernetes-client` and its `AbstractKubernetesTestResource` which already set fabric8 systemProps and dev services may interfere. Moreover, from my understanding, `@QuarkusDevModeTest` start the apps in Dev Mode after dev services already starts in Test mode as part of the buildStep.